webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -150,166 +153,147 @@ class ManageManifestModal extends Component {
-                          />
-                        </Col>
-                      </FormGroup>
-                    </React.Fragment>
-                  }
-                </Form>
-              </Tab>
-            }
-            <Tab eventKey={2} title={__('Manifest History')}>
-              <LoadingState loading={manifestHistory.loading} loadingText={__('Loading')}>
-                <Table
-                  rows={manifestHistory.results}
-                  columns={columns}
-                  emptyState={emptyStateData()}
-                />
-              </LoadingState>
+                          {canDeleteManifest &&
+                          <React.Fragment>
+                            <TooltipButton
+                              renderedButton={(
+                                <Button
+                                  disabled={!manifestExists(organization) || actionInProgress}
+                                  bsStyle="danger"
+                                  onClick={this.showDeleteManifestModal}
+                                >
+                                  {__('Delete')}
+                                </Button>
+                                )}
+
+                              tooltipId="delete-manifest-button-tooltip"
+                              tooltipText={this.disabledTooltipText()}
+                              tooltipPlacement="top"
+
+                            />
+                          </React.Fragment>
+                          }
+                        </div>
+                        <ForemanModal title={__('Confirm delete manifest')} id={DELETE_MANIFEST_MODAL_ID}>
+                          <ForemanModal.Header />
+                          <DeleteManifestModalText />
+                          <ForemanModal.Footer>
+                            <Button bsStyle="default" onClick={this.hideDeleteManifestModal}>
+                              {__('Cancel')}
+                            </Button>
+                            <Button bsStyle="danger" onClick={() => this.deleteManifest()}>
+                              {__('Delete')}
+                            </Button>
+                          </ForemanModal.Footer>
+                        </ForemanModal>
+                      </Col>
+                    </FormGroup>
+                  </React.Fragment>
+                }
+              </Form>
             </Tab>
-          </Tabs>
-        </Modal.Body>
-        <Modal.Footer>
+          }
+          <Tab eventKey={2} title={__('Manifest History')}>
+            <LoadingState loading={manifestHistory.loading} loadingText={__('Loading')}>
+              <Table
+                rows={manifestHistory.results}
+                columns={columns}
+                emptyState={emptyStateData()}
+              />
+            </LoadingState>
+          </Tab>
+        </Tabs>
+        <ForemanModal.Footer>
           <Button bsStyle="primary" onClick={this.hideModal}>
```

Comment:
Why should it hide both modals if this button is only accessible from the `Manage Manifest` ?

Reply:
@MariaAga It was just for peace of mind mostly.  Wanted to guarantee that there would never be a state where the delete confirmation dialog is showing but the ManageManifestModal is not.

Reply:
Makes sense